packages/kit/src/runtime/client/client.js

```diff
@@ -1236,9 +1249,10 @@ export function create_client({ target, base }) {
 			observer.disconnect();
 
 			for (const a of target.querySelectorAll('a')) {
-				const { url, external, options } = find_anchor(a, base);
+				const { url, external } = get_link_info(a, base);
+				const options = get_router_options(a);
```

Comment:
```suggestion
				if (external) continue;

				const options = get_router_options(a);
				if (options.reload) continue;
```

We should not get the router options if the link is external anyway (same logic as in the `preload` function.).